Plugin.php

```diff
@@ -48,4 +48,14 @@ public function registerNotificationRules()
             ],
         ];
     }
+
+    public function registerPermissions()
+    {
+        return [
+            'rainlab.notify.manage_notifications' => [
+                'tab' => SettingsManager::CATEGORY_NOTIFICATIONS,
+                'label' => 'Notifications management'
```

Comment:
This is the permission string I meant that needs to be translated.

Reply:
Sorry. I did not immediately understand.